docs/docs/reference-react.md

```diff
@@ -65,6 +65,12 @@ Validators treat props as optional by default. You can use `isRequired` to make
 
  - [`isRequired`](#isrequired)
 
+### Add-Ons
+
+If you're using [React with Add-ons](/react/docs/addons.html), they will be avaible via `React.addons`.
+
+ - [`React.addons`](#react.addons)
+
```

Comment:
Maybe add a note here that if you use npm, they will never be on `React.addons`. It only exists in the `react-with-addons.js` single file build.